vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -655,6 +660,7 @@ - (void)onReconcileComplete:(ledger::Result)result viewingId:(const std::string
     if (category == ledger::REWARDS_CATEGORY::RECURRING_TIP) {
       [self showTipsProcessedNotificationIfNeccessary];
     }
+    [self fetchBalance:nil];
```

Comment:
Not sure if it matters but this might have to go _after_ `OnReconcileCompleteSuccess` is called below. (`OnReconcileCompleteSuccess` ends up setting balance report items, but not entirely sure if its needed for balance).

Have you tested with this and see if it works?

Reply:
Yup, balance is fetched from server and has no coupling with balance report or contribution info. I just test it again to validate.

Reply:
Cool, just making sure